src/trading/README.md

```diff
@@ -0,0 +1,284 @@
+ - `appDataInfo` - [order's metadata](https://docs.cow.fi/cow-protocol/reference/sdks/app-data)
+ - `orderTypedData` - EIP-712 typed data for signing
+
+Another parameter is returned by this function is `postSwapOrderFromQuote`.
+It can be used to create an order from the received quote.
+
+#### Example
+
+```typescript
+import {
+  SupportedChainId,
+  OrderKind,
+  TradeParameters,
+  TradingSdk
+} from '@cowprotocol/cow-sdk'
+
+const sdk = new TradingSdk({
+  chainId: SupportedChainId.SEPOLIA,
+  signer: '<privateKeyOrEthersSigner>',
+  appCode: '<YOUR_APP_CODE>',
+})
+
+const parameters: TradeParameters = {
+  kind: OrderKind.BUY,
+  sellToken: '0xfff9976782d46cc05630d1f6ebab18b2324d6b14',
+  sellTokenDecimals: 18,
+  buyToken: '0x0625afb445c3b6b7b929342a04a22599fd5dbb59',
+  buyTokenDecimals: 18,
+  amount: '120000000000000000'
+}
+
+const { quoteResults, postSwapOrderFromQuote } = await sdk.getQuote(parameters)
+
+const buyAmount = quoteResults.amountsAndCosts.afterSlippage.buyAmount
+
+if (confirm(`You will get at least: ${buyAmount}, ok?`)) {
+  const orderId = await postSwapOrderFromQuote()
+
+  console.log('Order created, id: ', orderId)
+}
+```
+
+### postLimitOrder
+
+This function is simpler than the `postSwapOrder` function, because it doesn't require a quote.
+
+You need to provide the following parameters:
+ - `kind` - the order kind (sell/buy)
+ - `sellToken` - the sell token address
+ - `sellTokenDecimals` - the sell token decimals
+ - `buyToken` - the buy token address
+ - `buyTokenDecimals` - the buy token decimals
+ - `sellAmount` - the amount to sell in atoms
+ - `buyAmount` - the amount to buy in atoms
+
+```typescript
+import {
+  SupportedChainId,
+  OrderKind,
+  LimitTradeParameters,
+  TradingSdk
+} from '@cowprotocol/cow-sdk'
+
+const sdk = new TradingSdk({
+  chainId: SupportedChainId.SEPOLIA,
+  signer: '<privateKeyOrEthersSigner>',
+  appCode: '<YOUR_APP_CODE>',
+})
+
+const limitOrderParameters: LimitTradeParameters = {
+  kind: OrderKind.BUY,
+  sellToken: '0xfff9976782d46cc05630d1f6ebab18b2324d6b14',
+  sellTokenDecimals: 18,
+  buyToken: '0x0625afb445c3b6b7b929342a04a22599fd5dbb59',
+  buyTokenDecimals: 18,
+  sellAmount: '120000000000000000',
+  buyAmount: '66600000000000000000',
+}
+
+const orderId = await sdk.postLimitOrder(limitOrderParameters)
+
+console.log('Order created, id: ', orderId)
+```
+
+### postOnChainTrade
+
+CoW Protocol supports on-chain trades for selling blockchain native tokens (ETH for Ethereum).
+In this case, the order is created as an on-chain transaction.
+You don't have to think about the case when you use `postSwapOrder` function, it will be handled automatically.
+But if you need more flexible way to create an on-chain order, you can use the `postOnChainTrade` function.
+
+> We consider the order as an on-chain trade if the sell token has '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee' address.
```

Comment:
How about creating a preSign order?
That's is also an onchain trade. I know that exists and for me it's confusing to call only ethflow orders onchain trades.

Reply:
You 100% right.
Renamed to `postSellNativeCurrencyTrade`.
https://github.com/cowprotocol/cow-sdk/pull/213/commits/6739150b7ad4fb6dc87e7a29a5a49fc889fba69f